components/input-number/InputNumber.razor.cs

```diff
@@ -59,6 +59,9 @@ public TValue DefaultValue
         [Parameter]
         public TValue Min { get; set; }
 
+        [Parameter]
+        public int MaxLength { get; set; } = -1;
```

Comment:
Please set the default value to `null` so that the `maxlength` attribute would not be generated by default.

Reply:
Thanks for your reply! Updated in the latest commit.

Reply:
The property type also ned to be nullable `int?`